theano/scalar/basic.py

```diff
@@ -3099,7 +3109,8 @@ def init_name(self):
                 for i, r in enumerate(self.fgraph.variables):
                     if r not in io and len(r.clients) > 1:
                         r.name = 't%i' % i
-                rval = "Composite{%s}" % str(self.fgraph)
+                rval = "Composite{%s}" % ', '.join([pprint(output) for output
+                                                    in self.fgraph.outputs])
```

Comment:
I'm fine with this, but if there is many outputs, this hide the reuse of the different part. Just a comment so that if you see this, we know it is just they way it is printed.

I looked at the code of pprint() and it would accept this: pprint(self.fgraph.outputs). This would remove this "problem".